terraform/modules/app-ecs-services/templates/alertmanager.tpl

```diff
@@ -153,7 +161,8 @@ receivers:
       url: '{{ .CommonAnnotations.runbook_url }}'
 - name: "autom8-gsp-alerts-slack"
   slack_configs:
-  - send_resolved: true
+  - &GSP-slack-config
```

Comment:
Can we lowercase this for consistency with the rest of the _thing_ names? (eidas, autom8, dcs)